src/sync/pull-request.ts

```diff
@@ -87,9 +98,19 @@ export default async function(
 	const pullRequests = (
 		await Promise.all(
 			edges.map(async (pull) => {
+				const prDetails = useNewGHClient ? await newGithub.getPullRequest(repository.owner.login, repository.name, pull.number) :
+					(await github?.pulls?.get({
+						owner: repository.owner.login,
+						repo: repository.name,
+						pull_number: pull.number
+					})).data;
+
+				//TODO: Add getGithubUser call to new GH client
+				const ghUser = await getGithubUser(github, prDetails.user?.login);
 				const data = await transformPullRequest(
 					{ pullRequest: pull, repository },
-					github
```

Comment:
So did we pass a client into transformPullRequest before? and now we extract that part here?